src/area.c

```diff
@@ -12,8 +12,27 @@
 
 static const char *MODULE = "Area";
 
-// area-mouse-event.c
-napi_value create_mouse_event(napi_env env, uiAreaMouseEvent *event);
+static napi_ref AreaMouseEvent;
+
+napi_value create_mouse_event(napi_env env, uiAreaMouseEvent *e) {
+	napi_value global, constructor, value;
```

Comment:
not need anymore?

Reply:
Do you mean removing the function or the global variable?

Reply:
the global variable, it is not used anywhere

Reply:
Done